src/test/resources/pages/instructorFeedbacksAllSessionTypesWithHelperView.html

```diff
@@ -467,7 +467,7 @@ <h5 class="col-sm-2">
urseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" title="View the submitted responses for this feedback session" data-toggle="tooltip" data-placement="top">View Results</a> <a class="btn btn-default btn-xs btn-tm-actions session-edit-for-test" href="/page/instructorFeedbackEditPage?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" title="Edit feedback session details" data-toggle="tooltip" data-placement="top">Edit</a> <a class="btn btn-default btn-xs btn-tm-actions session-delete-for-test" href="/page/instructorFeedbackDelete?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;next=%2Fpage%2FinstructorFeedbacksPage%3Fuser%3DCFeedbackUiT.helper&amp;user=CFeedbackUiT.helper" title="Delete the feedback session" data-toggle="tooltip" data-placement="top" onclick="return toggleDeleteFeedbackSessionConfirmation('CFeedbackUiT.CS1101','Copied Session');">Delete</a> <a class="btn btn-default btn-xs btn-tm-actions session-submit-for-test" href="/page/instructorFeedbackSubmissionEditPage?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" title="Start submitting feedback" data-toggle="tooltip" data-placement="top">Submit</a> <div class="btn-group"><a class="btn btn-default btn-xs btn-tm-actions session-remind-for-test disabled" onclick="return false" href="/page/instructorFeedbackRemind?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" title="Send e-mails to remind students and instructors who have not submitted their feedbacks to do so" data-toggle="tooltip" data-placement="top">Remind</a><button type="button" class="btn btn-default btn-xs btn-tm-actions dropdown-toggle session-remind-options-for-test" data-toggle="dropdown" aria-expanded="false"> <span class="caret"></span></button> <ul class="dropdown-menu" role="menu"><li><a href="/page/instructorFeedbackRemind?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" class="session-remind-inner-for-test" title="Send e-mails to remind students and instructors who have not submitted their feedbacks to do so" data-toggle="tooltip" data-placement="top">Remind all students</a></li><li><a href="#" data-actionlink="/page/instructorFeedbackRemindParticularStudentsPage?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;user=CFeedbackUiT.helper" class="session-remind-particular-for-test" data-courseid="CFeedbackUiT.CS1101" data-fsname="Copied Session" data-toggle="modal" data-target="#remindModal">Remind particular students</a></li></ul></div> <a class="btn btn-default btn-xs btn-tm-actions session-publish-for-test disabled" onclick="return false" href="/page/instructorFeedbackPublish?courseid=CFeedbackUiT.CS1101&amp;fsname=Copied+Session&amp;next=%2Fpage%2FinstructorFeedbacksPage%3Fuser%3DCFeedbackUiT.helper&amp;user=CFeedbackUiT.helper" title="This session is not yet opened" data-toggle="tooltip" data-placement="top">Publish</a> </td>
```

Comment:
this line is different. remove the disabled for caret. adding disabled for links in drop down instead